### DISCUSSION TITLE
Tuần 4

### DISCUSSION THREAD

**Discussion by dangxuantruong10a13** [dangxuantruong10a13/nhom9_10a13#32] [General]
Mn ơi phần về bản thân mình thì tui sẽ thêm về các việc mà mình đã làm trong cuộc sống này nó thể hiện sự tử tế đúng k mn, có cần thêm mình tôn trọng cách mà những người đi trước đã làm nhiều hành động thể hiện sự tử tế trong cuộc sống k (tôn trọng những người trong dẫn chứng của Lâm). Gấp nka ;v

**Comment by 04lananh** [comment]
Đúng òi nha bản thân tôi thì nói về những việc mình đg lm đã lm và sẽ làm á

**Comment by 04lananh** [comment]
Chủ yếu là liên hệ bản thân thui

**Comment by dangxuantruong10a13** [comment]
Là lấy bản thân làm dẫn chứng như hồi lớp 9 hay làm á hả, oke hên quá em làm đúng r kkkkkk. Giờ để lấy thêm mấy cái bài báo liên hệ chính tác giả đêt tham khảo xong update bài quăng lên code lun là xong tuần 4 kêkkekekekekekekeke

**Comment by dangxuantruong10a13** [comment]
Lan Anh có hỏi gì hong ák cách làm word hay j đại loại nek

**Comment by NgHan12pn** [comment]
Ủa h mình làm word lun hả

**Reply by dangxuantruong10a13** [reply]
đúng rồi bà

**Comment by dangxuantruong10a13** [comment]
tui up phần bản thân tôi lên issues rồi đó mn có đọc và cho tui nhận xét nha

**Reply by TranBaoLam10C13** [reply]
Oke

**Comment by dangxuantruong10a13** [comment]
ok vậy là đc r tuần sau mình sẽ hoàn thành hết và kết thúc chuỗi ngày này nha kkkkkk. Mn chỉnh word lại hết mấy bạn mà tuần này vẫn chưa xong việc ý. tuần sau sẽ gửi tui r tổng hợp lại trang trí các thứ nke

**Reply by TranBaoLam10C13** [reply]
Haiiiii!

**Reply by NgHan12pn** [reply]
Ê làm word r kĩ thuật j nữa là sao ???

**Reply by NgHan12pn** [reply]
Chỉ tui cách làm vớiiii

**Reply by dangxuantruong10a13** [reply]
là mình thêm định dạng chữ nghiêng chữ dọc, art work, các kiểu ák pà

**Reply by TranBaoLam10C13** [reply]
![2615D1E4-9553-44AA-B62C-E0F46C222A28](https://user-images.githubusercontent.com/91810620/158065273-b68b557e-0dcd-4884-9b66-a31dfe310947.jpeg)

**Reply by dangxuantruong10a13** [reply]
thì mình sẽ cho câu nói là chữ nghiêng là đc r :)). mấy cái muốn nhấn mạnh thì in đậm

**Reply by dangxuantruong10a13** [reply]
à đối với mở bài thoi nka

**Reply by NgHan12pn** [reply]
Có chú thích mấy cái kĩ thuật như thầy nói kh

**Reply by TranBaoLam10C13** [reply]
Tổng hợp lm luôn

**Reply by NgHan12pn** [reply]
Okayy tks nhìuuu

**Reply by TangVinhAn-PNc13** [reply]
ok bro
